ui/src/pages/help-center/HelpCenter.tsx

```diff
@@ -0,0 +1,36 @@
+import { Box } from '@mui/material';
+import { Page } from 'components/page';
+import { FC, useCallback } from 'react';
+import { Messages } from './HelpCenter.messages';
+import { CardsData } from './HelpCenter.constants';
+import { useUser } from 'contexts/user';
+import { HelpCenterCard } from './help-center-card/HelpCenterCard';
+
+export const HelpCenter: FC = () => {
+  const { user } = useUser();
+
+  const shouldDisplayCard = useCallback(
```

Comment:
You can move this into the `HelpCenterCard` component like so:

```tsx
const HelpCenterCard: FC = ({ ...,card }) => {
  const { user } = useUser();
  const { adminOnly, ... } = card;
  
  if (adminOnly && !user.isPMMAdmin) {
    return null;
  }
}
```
Doing the check this way would prevent the additional passing of the `shouldDisplayCard` function down to the card components.


Another option (preffered) is to filter out the cards like so:

```ts
const { user } = useUser(); 
const cards = CARDS_DATA.filter(card => !card.adminOnly || user.isPMMAdmin);

// render cards as usual
```

This is probably a bit cleaner since it will not render nulls in place of the cards if the permissions don't match.